src/imvujstest/assert.js

```diff
@@ -19,7 +19,7 @@ module({
         if (typeof selectorOrJQueryObject === 'string') {
             return 'Selector ' + formatTestValue(selectorOrJQueryObject);
         } else if (typeof selectorOrJQueryObject === 'object') {
-            return "'" + selectorOrJQueryObject[0] + "'";
```

Comment:
I could've also renamed `selectorOrJQueryObject` to `elementOrSelectorOrJQueryObject` for this change. Should I do that?